February 2023 newsletter (#352)
readthedocs/ethicalads.io: content/posts/newsletter-february-2023.md

Title: EthicalAds Newsletter - February 2023
Date: March 7, 2023
description: Updates from February 2023, including stats and commentary on our progress as we build EthicalAds
tags: newsletter, community, build-in-public
authors: David Fischer
image: /images/posts/2023-february-newsletter.jpg
image_credit: <span>Photo by <a href="https://unsplash.com/@blue_jean?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Izzy Park</a> on <a href="https://unsplash.com/photos/uLrwuNznzO0?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a></span>

## New features

> These are the major features we released in February:

Reviewer: ```suggestion
Our progress in February was:
```

We say `released` a lot in this post. We should either remove it here, or remove it from each of the line items below.

* We released a number of performance improvements to our reporting.
  All timeout issues have been resolved and performance is downright snappy on most reports.

> * We released some improvements to our ad pacing that we believe will yield
>   slightly better performance for our advertisers.

Reviewer: ```suggestion
* We released some improvements to our ad pacing that will yield
  better performance for our advertisers. Specifically we now allow pacing by the hour instead of by the day to spread out ads across the day more effectively. 
```

> * Also in our server, we released a change that will prioritize ads
>   with slightly better performance. This is another quality of life change for advertisers
>   in order to improve their performance.

Reviewer: ```suggestion
* We released a change that will prioritize ads
  with high CTRs. This is another quality of life change for advertisers
  in order to improve their performance.
```

> * A common question we get from advertisers is around measuring conversions.

Reviewer: This is a weird shift. I don't want to overly copy edit, but I think it makes sense to use the format of `<thing we shipped, explanation of it>`

Author: I reworked it a bit

  Advertisers are always measuring how much they spent on ads and how many conversions they received.
  We [documented a few tips on measuring conversions]({filename}../pages/learning-hub/advertiser-guide.md#measuring-conversions)
  but we always want our ad network to be measured
  the same way against others so advertisers can see for themselves that our network works well.
* We built a number of internal sales dashboards to better measure our inventory,
  what's starting soon and what is outstanding. The hope here is to be able to bring
  in some additional hands for our small sales team.

You can always see our latest server updates in our [ethical-ad-server changelog](https://ethical-ad-server.readthedocs.io/en/latest/developer/changelog.html) and [ethical-ad-client changelog](https://ethical-ad-client.readthedocs.io/en/latest/changelog.html).

## February advertising stats

[comment]: https://server.ethicalads.io/publisher/all/report/?start_date=2023-02-01&end_date=2023-02-28

In February 2023:

* We generated **$34,464.67** for our publishers.

> * We had **147 publishers** on our network with at least one paid ad impression.

Reviewer: This seems higher than before! I assume it's been creeping up, but should be over 150 here soon.

Author: It is actually the same as the previous month but it has been creeping up.

* We had **12,773,611** paid ad views across the world.

As we mentioned last month, we expected February to be a bit stronger than January
and we generated about $2,500 in additional revenue for our publishers despite the shorter month.

> We expect March to be an even stronger still.

Reviewer: ```suggestion
We expect March to be even stronger still.
```
